airbyte-cdk/python/bin/run-cats-with-local-cdk.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env sh
+
+ROOT_DIR="$(git rev-parse --show-toplevel)"
+OUTPUT_DIR=/tmp/cat-output
+SCRIPT=/tmp/run-cats-with-local-cdk.sh
+# Clean up from previous test runs
+rm -rf $OUTPUT_DIR && mkdir $OUTPUT_DIR
+
+while getopts ":c:" opt; do
```

Comment:
would be useful to give an example of how to run the script

Reply:
👍 added help text.